tests/Rules/Data/relation-existence-rule.php

```diff
@@ -7,6 +7,7 @@
 \App\User::query()->doesntHave('foo');
 \App\User::query()->orDoesntHave('foo');
 \App\User::query()->whereHas('foo');
+\App\User::query()->withWhereHas('foo');
```

Comment:
Please add this to the end of the file. Also the others. So that way you don't need to change any line number in  `tests/Rules/RelationExistenceRuleTest.php`